opssat-package/src/main/resources/space-supervisor-opssat-root/settings.properties

```diff
@@ -30,3 +30,6 @@ nmf.supervisor.parameter.valuesprovider.nanomind.maxQueryRate=10
 
 # interval (seconds) at which we clean parameters not used anymore from aggregations in the Nanomind
 nmf.supervisor.parameter.valuesprovider.nanomind.cleaningInterval=300
+
+# parameter ID filter (allow SEPTM parameters -> 2242 - 2278 from Datapool.xml)
+nmf.supervisor.parameter.valuesprovider.nanomind.allowedParamIDs=2242-2278
```

Comment:
rename the property to indicate clearly that these are params allowed for *writing* - possibly `writableParameters`?